apps/examples/testcase/le_tc/kernel/tc_libc_timer.c

```diff
@@ -415,6 +415,108 @@ static void tc_libc_timer_strftime(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+* @fn             :tc_libc_timer_strptime
+* @brief          :strptime - Format string as time
+* @Scenario       :strptime - converts time string as the time described in timeptr
+*                  structure as per the format specifiers.
+* API's covered   :strptime
+* Preconditions   :none
+* Postconditions  :none
+* @return         :void
+*/
+static void tc_libc_timer_strptime(void)
+{
+	struct tm sp_tm;
+	struct tm sf_tm;
+	char *ret;
+
+	sf_tm.tm_sec  = 13;
+	sf_tm.tm_min  = 42;
+	sf_tm.tm_hour = 12;
+	sf_tm.tm_mday = 5;
+	sf_tm.tm_mon  = 5;
+	sf_tm.tm_year = 2018 - YEAR_BASE;
+
+	memset(&sp_tm, 0, sizeof(struct tm));
+	ret = strptime("2018-06-05 12:42:13 PM", "%Y-%m-%Od %k:%M:%S %p", &sp_tm);
```

Comment:
I think it would be great, if you can add comments before these operations so that developers or reviewers can understand what exactly strptime is doing in this case and what is expected output.

because all testcases have strptime usage but with different usecase.